<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-03T12:08:13 SAST" name="Suite" finished-at="2025-09-03T12:09:52 SAST" duration-ms="99168">
    <groups>
      <group name="Regression">
        <method signature="Reporting.generateAllReportsAndAssertStatusMessages()[pri:0, instance:UtilipayV2Hybrid.testCases.Reporting@3016fd5e]" name="generateAllReportsAndAssertStatusMessages" class="UtilipayV2Hybrid.testCases.Reporting"/>
      </group> <!-- Regression -->
    </groups>
    <test started-at="2025-09-03T12:08:13 SAST" name="Utilipay Version 2" finished-at="2025-09-03T12:09:52 SAST" duration-ms="99168">
      <class name="UtilipayV2Hybrid.testCases.Reporting">
        <test-method is-config="true" signature="setUp(java.lang.String,java.lang.String)[pri:0, instance:UtilipayV2Hybrid.testCases.Reporting@3016fd5e]" started-at="2025-09-03T12:08:13 SAST" name="setUp" finished-at="2025-09-03T12:08:17 SAST" duration-ms="3835" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[prepaid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="setupReportingModule()[pri:0, instance:UtilipayV2Hybrid.testCases.Reporting@3016fd5e]" started-at="2025-09-03T12:08:17 SAST" name="setupReportingModule" finished-at="2025-09-03T12:08:20 SAST" duration-ms="2446" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupReportingModule -->
        <test-method retried="true" signature="generateAllReportsAndAssertStatusMessages()[pri:0, instance:UtilipayV2Hybrid.testCases.Reporting@3016fd5e]" started-at="2025-09-03T12:08:20 SAST" name="generateAllReportsAndAssertStatusMessages" finished-at="2025-09-03T12:09:07 SAST" duration-ms="47301" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Unexpected status message for report: Automation Report expected [Your report is being prepared. you will receive an email ones it is ready!] but found [Preset successfully saved!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Unexpected status message for report: Automation Report expected [Your report is being prepared. you will receive an email ones it is ready!] but found [Preset successfully saved!]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at UtilipayV2Hybrid.testCases.Reporting.generateAllReportsAndAssertStatusMessages(Reporting.java:99)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateAllReportsAndAssertStatusMessages -->
        <test-method signature="generateAllReportsAndAssertStatusMessages()[pri:0, instance:UtilipayV2Hybrid.testCases.Reporting@3016fd5e]" started-at="2025-09-03T12:09:07 SAST" name="generateAllReportsAndAssertStatusMessages" finished-at="2025-09-03T12:09:52 SAST" duration-ms="44792" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Unexpected status message for report: Automation Report expected [Your report is being prepared. you will receive an email ones it is ready!] but found [Preset successfully saved!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Unexpected status message for report: Automation Report expected [Your report is being prepared. you will receive an email ones it is ready!] but found [Preset successfully saved!]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at UtilipayV2Hybrid.testCases.Reporting.generateAllReportsAndAssertStatusMessages(Reporting.java:99)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateAllReportsAndAssertStatusMessages -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:UtilipayV2Hybrid.testCases.Reporting@3016fd5e]" started-at="2025-09-03T12:09:52 SAST" name="tearDown" finished-at="2025-09-03T12:09:52 SAST" duration-ms="299" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- UtilipayV2Hybrid.testCases.Reporting -->
    </test> <!-- Utilipay Version 2 -->
  </suite> <!-- Suite -->
</testng-results>
